docs/research.md

```diff
@@ -72,11 +82,37 @@ Source:
 
 We have decided to develop our own CLI and infrastructure tooling to create a more flexible deployment process and address some of the inconveniences we encountered while testing existing solutions. Given the challenges in getting current solutions to work easily and the relatively short time required to replicate them, we aim for our development to also meet new requirements, including a user interface and scalable deployment.
 
-## Decisions and drivers
+# Decisions and drivers
+
+We thrive to make things the best way possible, our tech decisions are focused on popularity, usability, simplicity and efficiency.
+
+## Technology
+
+#### Rust
+
+It's increasing popularity, long-term maintainability and suitability for the task.
+
+#### Kubernetes & Helm
+
+Taking a step on top of docker-compose, Kubernetes with Helm allow for scalability and simplify resource management.
+
+#### HashiCorp's Terraform
+
+Terraform is a powerful IaC tool that simplifies the deployment-provider relationship.
+It works through the generation of agnostic resource templates for their deployment on different providers with minimal changes.
+
+#### AWS
+
+We use AWS as an example provider to deploy to.
+The web services provider may be up to the developr to chose, shifting using Terraform should ease the way.
+
+#### Github actions
+
+CI/CD integrated on the same repository, it uses yaml files to make automated pipelines and drive new releases delivery. 
```

Comment:
I think I would remove all this, doesn't add much value and at this point doesn't have much context, the reasons why we use them are below. Also would remove focus from rust as is not attached to the core problem, docker-compose as it's just a resource and aws as we're aiming to be open for other providers...

Reply:
Gotcha, makes sense to me